contracts/protocol/tokenization/NTokenUniswapV3.sol

```diff
@@ -107,44 +109,73 @@ contract NTokenUniswapV3 is NToken {
             );
         }
 
+        (
+            ,
+            ,
+            address token0,
+            address token1,
+            ,
+            ,
+            ,
+            ,
+            ,
+            ,
+            ,
+
+        ) = INonfungiblePositionManager(_underlyingAsset).positions(tokenId);
+
+        address weth = _addressesProvider.getWETH();
```

Comment:
we start to have multiple cases for weth wording, I think we can be consistent and use only:

WETH because it's a keyword

instead of having:

- weth
- Weth
- WETH

Reply:
I think it's ok, we can have the style like this.
```
    IWETH internal immutable WETH;
    address public immutable weth;
```